.rubocop.yml

```diff
@@ -388,6 +391,12 @@ Rails/DynamicFindBy:
     - find_by_slug_or_id!
     - find_by_manager_login
 
+Rails/EnumHash:
```

Comment:
Shall we squash this commit and the previous one? That way the rule is added and applied in the same commit :thinking:.